stewart/templates/header-footer-only.html

```diff
@@ -0,0 +1,10 @@
+<!-- wp:template-part {"slug":"header","tagName":"header"} /-->
+
+<!-- wp:group {"tagName":"main"} -->
+<main class="wp-block-group">
+
+<!-- wp:post-content {"layout":{"inherit":true}} /-->
+</main>
+<!-- /wp:group -->
+
+<!-- wp:template-part {"slug":"footer","tagName":"footer","layout":{"inherit":true},"className":"site-footer-container"} /-->
```

Comment:
I think we should remove this class name too.

Reply:
then we'll be missing the "sticky footer" styles https://github.com/Automattic/themes/blob/75542ea9cbaade68bc827b353bd8e23c8c69e8f8/blockbase/sass/base/_layout.scss#L12

Reply:
This isn't a Blockbase child